oc2ls.md

```diff
@@ -2226,6 +2258,8 @@ requested in the Command.
     "status": 102
 }
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+              
+*Editor's Note: Replace with an example that does not use "properties".*
```

Comment:
PR #401 deletes this example entirely. Some other example may be worth adding in its place.